Replicator/Replicator.cc

```diff
@@ -48,7 +50,8 @@ namespace litecore { namespace repl {
     static constexpr StoppingErrorEntry StoppingErrors[] = {
         {{ LiteCoreDomain, kC4ErrorUnexpectedError,0 }, true, "An exception was thrown"_sl},
         {{ WebSocketDomain, 403, 0}, true, "An attempt was made to perform an unauthorized action"_sl},
-        {{ WebSocketDomain, 503, 0 }, false, "The server is over capacity"_sl}
+        {{ WebSocketDomain, 503, 0 }, false, "The server is over capacity"_sl},
+        {{ LiteCoreDomain, 26, 1 }, true, "No handler for BLIP request 'getCollections'"_sl }
```

Comment:
Better to use kC4ErrorRemoteError instead of 26.

Reply:
I also suggest changing the message here, if you think keeping 26 is the right call.  You can log a context specific message before you get to this point, but if this is the message here in the stopping error list then it's going to log this message here regardless of the cause or context.

Reply:
I strongly agree.

Reply:
^^ I've done that now, on latest commit